apps/api/src/app/notification-template/usecases/update-notification-template/update-notification-template.usecase.ts

```diff
@@ -1,5 +1,6 @@
 // eslint-ignore max-len
 import { BadRequestException, Inject, Injectable, NotFoundException } from '@nestjs/common';
+import { AnyKeys } from 'mongoose';
```

Comment:
Unfortuently we should not use `mongoose` related things in API. Can we re export this thing from the dal package? Ideally we would want the application to know as less as possible about mongoose or mongo

Reply:
There is a small difference, `Partial` still requires the correct types for values to be passed, but the `AnyKeys` allows either for the correct value type or for any other type. So for this file it might work, but I've used it in another place when we do assign the `_feedId: null` but the type is `string | undefined`, so it raises the issue there, but yeah I'll fix that place too.